prelude/http_archive/http_archive.bzl

```diff
@@ -175,24 +170,32 @@ def http_archive_impl(ctx: AnalysisContext) -> list[Provider]:
         mkdir = "mkdir -p {}"
         interpreter = ["/bin/sh"]
 
-    unarchive_cmd = _unarchive_cmd(ext_type, exec_is_windows, archive, ctx.attrs.strip_prefix)
+    unarchive_cmd, was_prefix_kept = _unarchive_cmd(ext_type, exec_is_windows, archive, ctx.attrs.strip_prefix)
```

Comment:
Is `needs_strip_prefix` a better name than `was_prefix_kept` - since if the prefix was kept because you didn't ask the prefix to be stripped it returns `False`, but the prefix was kept.

Reply:
That is definitely a better name